esmvaltool/recipes/recipe_landcover.yml

```diff
@@ -29,7 +29,7 @@ documentation:
 
 datasets:
   - {dataset: MPI-ESM-LR,       project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
-  - {dataset: inmcm4,           project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
+  #- {dataset: inmcm4,           project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
```

Comment:
I'm not the right person to decide. If the recipe was used for some publications, I think it would be better to comment out the faulty models (and keep that for reference) than just deleting them from the recipes.

Reply:
@axel-lauer what do you think?